sdk/trace/trace_test.go

```diff
@@ -602,7 +602,15 @@ func cmpDiff(x, y interface{}) string {
 	return cmp.Diff(x, y,
 		cmp.AllowUnexported(core.Value{}),
 		cmp.AllowUnexported(export.Event{}),
-		cmp.AllowUnexported(resource.Resource{}))
+		cmp.Comparer(func(a, b *resource.Resource) bool {
+			if a == nil && b == nil {
+				return true
+			} else if a != nil && b != nil {
+				return a.Labels().Equivalent() == b.Labels().Equivalent()
```

Comment:
Possibly use the convenience method?

```suggestion
				return a.Labels().Equal(b.Labels())
```

Reply:
I was able to remove this code from the test.  The problem was when some code is using a pointer and other code is using a struct, that the special support from `cmp` was needed.

I have added a `Set.Equal()` helper method anyway.